dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -712,6 +712,10 @@ error.invalidPhaseName=CWWKE0954E: The "{0}" checkpoint phase specified is empty
 error.invalidPhaseName.explanation=An unknown phase was specified as the value of the checkpoint option.
 error.invalidPhaseName.useraction=Review the documentation to determine valid phase names. Reissue the command using a valid checkpoint phase.
 
+error.set.securitymanager.jdk18=CWWKE0955E: The property "websphere.java.security" was set in the bootstrap.properties file while using Java "{0}". This property enables the Java Security Manager and is no longer valid when using Java 18 or later.
```

Comment:
recommend removing the quotation marks here and in the useraction and putting the noun identifier (property) after the property name. Also, avoid elliptical verb constructions like "while using" and they can be difficult for translators.

The websphere.java.security property was set in the bootstrap.properties file but the Java version is Java {0}. This property enables the Java Security Manager and is not valid with Java 18 or later.

Reply:
Gotcha.  I had already delievered just the message changes with https://github.com/OpenLiberty/open-liberty/pull/20305 as this issue may not be merged before the March 11 deadline.  I'll create another PR and apply your recommendations.